docs/topics/Response_Codes.md

```diff
@@ -42,6 +42,7 @@ Along with the HTTP error code, our API can also return more detailed error code
 | 10011 | Unknown role |
 | 10012 | Unknown token |
 | 10013 | Unknown user |
+| 10014 | Unknown Emoji |
```

Comment:
You are missing some response codes:

Too many reactions, 30010
Reaction blocked, 90001